applications/openshift/master/file_groupowner_ovn_db_files/tests/ocp4/e2e.yml

```diff
@@ -0,0 +1,3 @@
+---
+default_result: PASS
```

Comment:
This rules results in `NOT-APPLICABLE` due to `ovn` platform:
https://github.com/ComplianceAsCode/content/blob/9613cee7adba0d786e6b9131ef2447394362a153/applications/openshift/master/file_groupowner_ovn_db_files/rule.yml#L4